src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/LeggTilEllerFjernBrevmottakere.tsx

```diff
@@ -3,13 +3,22 @@ import React, { useState } from 'react';
 import { Dropdown } from '@navikt/ds-react';
 
 import { LeggTilBrevmottakerModalBehandling } from './LeggTilBrevmottakerModalBehandling';
+import { LeggTilBrevmottakerModalFagsak } from './LeggTilBrevmottakerModalFagsak';
+import type { SkjemaBrevmottaker } from './useBrevmottakerSkjema';
 import type { IBehandling } from '../../../../../typer/behandling';
 
 interface BehandlingProps {
+    erPåBehandling: true;
     behandling: IBehandling;
     erLesevisning: boolean;
 }
 
+interface FagsakProps {
+    erPåBehandling: false;
+    erLesevisning?: never;
```

Comment:
kunne denne ha vært false i stedet for never? Så slipper man !! lenger ned her

Reply:
Hmmm, den kan jo være det men da forsvinner litt av typingen. Når den er never her så får man feilmelding dersom man prøver å sende inn erLesevisning som prop, som jo blir riktig fordi man ikke skal sende inn erLesevisning når man er på fagsaknivå her 🤔

Reply:
Sant! Men en ting jeg kom på nå, burde man egentlig sjekke om saksbehandler har skrivetilgang og sende inn det resultatet som erLesevisning? Hva skjer hvis jeg som veileder prøver å gå inn her i prod 🤔 det gjelder forsåvidt andre ting i løsningen også som er på fagsaknivå (feks selve dokumentutsendingen)

Reply:
Saksbehandleren kommer seg bare inn på å sette manuell brevmottaker dersom man er på selve dokumentutsendingssteget. Så jeg tenker at det blir spørsmålet som du nevnte i siste setning, om de skal ha rettigheter til selve dokumentutsendingen. Jeg er usikker på hvordan det er i dag, men hvis vi skal endre det så er det kanskje en egen oppgave? 😊

Reply:
Ser at kallet i allefall blir stoppet i backend om man prøver å sende brev som noe annet enn saksbehandler